geometry_msgs/Pose pose
uint64 id
uint64[] map_point_ids
